schedule/schedule-appengine/src/test/java/com/jasify/schedule/appengine/util/FormatUtilTest.java

```diff
@@ -17,6 +19,20 @@
 
 public class FormatUtilTest {
 
+
+    @Before
+    public void before() {
+        // Your wondering why... well initializeDatastore sets the default timezone to UTC.
+        // FormatUtil.toString is called indirectly by other tests and the SimpleDateFormat is initialized with default timezone
+        // By the time we get here its UTC but your pc default timezone is probably not UTC so the tests will fail
+        TestHelper.initializeDatastore();
```

Comment:
I think this is the wrong fix for the case, relying on the default timezone to be UTC is probably not correct.  I can't understand what failed here so I might be wrong.